<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerfPage</class>
 <widget class="QWidget" name="PerfPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>639</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>639</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Спектакли и роли</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(113, 37, 37, 1); color: #fdefc8; font-family: 'Martian Mono';</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="performancesFrame">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(64, 20, 20, 1); border-radius: 15px; padding: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_1">
      <item>
       <widget class="QLabel" name="performancesLabel">
        <property name="styleSheet">
         <string notr="true">color: #fdefc8; font-size: 14px;</string>
        </property>
        <property name="text">
         <string>Спектакли</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="performancesList">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(50, 15, 15, 1); border-radius: 15px; padding: 10px; color: #fdefc8; font-size: 12px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPerformanceBtn">
        <property name="styleSheet">
         <string notr="true">background-color: #8B4513; color: #fdefc8; border: none; border-radius: 15px; padding: 8px 15px; font-size: 12px;</string>
        </property>
        <property name="text">
         <string>+ Добавить спектакль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deletePerformanceBtn">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#b14446; color: #fdefc8; border: none; border-radius: 15px; padding: 8px 15px; font-size: 12px;</string>
        </property>
        <property name="text">
         <string>- Удалить спектакль</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="rolesFrame">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(64, 20, 20, 1); border-radius: 15px; padding: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1,0,0,0,0,0">
      <item>
       <widget class="QLabel" name="detailsTitle">
        <property name="styleSheet">
         <string notr="true">color: #fdefc8; font-size: 16px; padding: 5px;</string>
        </property>
        <property name="text">
         <string>Выберите спектакль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="coverLabel">
        <property name="minimumSize">
         <size>
          <width>272</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px dashed #8B4513; color: #8B4513; background-color: rgba(50, 15, 15, 1);border-radius: 0px;padding: 0px;margin: 0px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Нет обложки</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="detailsFrame">
        <property name="maximumSize">
         <size>
          <width>272</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(80, 25, 25, 1); border-radius: 15px; padding: 15px; margin: 5px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="performanceDate">
           <property name="styleSheet">
            <string notr="true">color: #fdefc8; font-size: 12px; padding: 2px;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="performanceDescription">
           <property name="styleSheet">
            <string notr="true">color: #fdefc8; font-size: 12px; padding: 2px;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rolesLabel">
        <property name="styleSheet">
         <string notr="true">color: #fdefc8; font-size: 14px; padding: 5px;</string>
        </property>
        <property name="text">
         <string>Роли в спектакле:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="rolesList">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(50, 15, 15, 1);  border-radius: 15px; padding: 10px; color: #fdefc8; font-size: 12px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="applyBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #2E8B57; color: #fdefc8; border: none; border-radius: 15px; padding: 8px 15px; font-size: 12px;</string>
        </property>
        <property name="text">
         <string>Подать заявку на роль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addRoleBtn">
        <property name="styleSheet">
         <string notr="true">background-color: #8B4513; color: #fdefc8; border: none; border-radius: 15px; padding: 8px 15px; font-size: 12px;</string>
        </property>
        <property name="text">
         <string>+ Добавить роль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteRoleBtn">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>253</red>
              <green>239</green>
              <blue>200</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>177</red>
              <green>68</green>
              <blue>70</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#b14446; color: #fdefc8; border: none; border-radius: 15px; padding: 8px 15px; font-size: 12px;</string>
        </property>
        <property name="text">
         <string>- Удалить роль</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="applicationsFrame">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(64, 20, 20, 1); border-radius: 15px; padding: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="applicationsLabel">
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #fdefc8; font-size: 14px; padding: 0px;</string>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Заявки на роли</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="applicationsList">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>430</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(50, 15, 15, 1); border-radius: 15px; padding: 10px; color: #fdefc8; font-size: 12px;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="approveBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #2E8B57; color: #fdefc8; border: none; border-radius: 15px; padding: 8px 15px; font-size: 12px;</string>
          </property>
          <property name="text">
           <string>✓ Одобрить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rejectBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #8B4513; color: #fdefc8; border: none; border-radius: 15px; padding: 8px 15px; font-size: 12px;</string>
          </property>
          <property name="text">
           <string>✗ Отклонить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
